testing/ncids-css-testing/stories/components/nci-card/default.stories.jsx

```diff
@@ -18,7 +18,7 @@ import img_news3_4x3 from './img/news3-4x3.jpg';
 const html = `
   <div id="nci-card" class="grid-container">
     <div class="nci-card tablet-lg:grid-col-4">
-      <a href="https://www.cancer.gov" aria-label="Feature Card">
+      <a href="javascript:void(0);" aria-label="Feature Card">
```

Comment:
do you try `href="#"`? certain versions of react will throw an error with `href="javascript:void(0);"`. If # works we could avoid the headache later. If not, we can worry about it then